ezcater_rubocop.gemspec

```diff
@@ -42,17 +42,17 @@ Gem::Specification.new do |spec|
   spec.executables << "circle_rubocop.rb"
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = ">= 2.6"
+  spec.required_ruby_version = ">= 2.7"
 
-  spec.add_development_dependency "bundler", "~> 2.1"
+  spec.add_development_dependency "bundler", "~> 2.3"
   spec.add_development_dependency "pry-byebug"
-  spec.add_development_dependency "rake", "~> 12.3"
-  spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency "rake", "~> 13.0"
+  spec.add_development_dependency "rspec", "~> 3.11"
   spec.add_development_dependency "rspec_junit_formatter"
   spec.add_development_dependency "simplecov", "< 0.18.0"
 
-  spec.add_runtime_dependency "parser", "!= 2.5.1.1"
-  spec.add_runtime_dependency "rubocop", "~> 1.16.0"
-  spec.add_runtime_dependency "rubocop-rails", "~> 2.10.1"
-  spec.add_runtime_dependency "rubocop-rspec", "~> 2.3.0"
+  spec.add_runtime_dependency "parser", "~> 3.1"
```

Comment:
This new constraint seems less permissive than the previous one. What is the requirement that pushed us to update this?

Reply:
Sure, I was playing with updated gems and ruby when I hit the problems. I can move those to `>=` of the previous versions.

Reply:
I made them more permissive.

Reply:
My concern is mostly about supporting a reasonable range so that other applications do not have to upgrade these gems when upgrading `ezcater_rubocop` unless ezcater rubocop is explicitly incompatible with the older versions.

Reply:
I agree. I set the dependencies to be >= the previous version. I'm looking for an app that uses the previous version and testing against that, but every app that I have worked on caps the version of this gem to really old versions. I will keep looking for a candidate rails app to test against.